api/types/cluster_alert.go

```diff
@@ -25,8 +25,11 @@ import (
 	"github.com/gravitational/trace"
 )
 
-// matchStrictLabel is a fairly conservative allowed charset for labels.
-var matchStrictLabel = regexp.MustCompile(`^[a-z0-9\.\-\/]+$`).MatchString
+// matchAlertLabelKey is a fairly conservative allowed charset for label keys.
+var matchAlertLabelKey = regexp.MustCompile(`^[a-z0-9\.\-\/]+$`).MatchString
```

Comment:
Are alert labels more strict than Teleport RBAC labels, or can we use the same validation in both cases?

Reply:
RBAC label values aren't checked at all, and RBAC label keys allow stuff like capitalization and wildcards. I prefer erring on the side of strictness, especially given the heavy use of custom alert labels. Keeps things more consistent. More of a personal preference than a real necessity though, so long as we continue to not display labels to users by default.